container.go

```diff
@@ -317,6 +339,7 @@ func (c *container) Update(ctx context.Context, opts ...UpdateContainerOpts) err
 	if _, err := c.client.ContainerService().Update(ctx, r); err != nil {
 		return errdefs.FromGRPC(err)
 	}
+	span.AddEvent("container updated")
```

Comment:
Do we need this event?

Reply:
hmm, now that I look at it, the span ending should be enough to know that the container is updated. Will remove this.